doc/guide/fitting_sq.rst

```diff
@@ -283,6 +283,11 @@ and the form:shell volume ratio as
 
     \phi_S = \phi_P \langle V_e \rangle \big/ \langle V_s \rangle
 
+.. note::
+
+    A change to how the shell volume is incorporated in the overall volume
+    normalisation was made in Sasmodels v1.0.5 in Nov 2020. The technical
+    details can be found `here <https://github.com/SasView/sasview/issues/1698#issuecomment-718979190>`_. 
```

Comment:
The "technical details" I provide in the link are too terse to be useful. How about:
```rst
Prior to Sasmodels v1.0.5 (Nov 2020), the intermediated $P(Q)$ returned by
the interaction calculator did not incorporate the volume normalization and
so $I(Q) \ne P(Q) S(Q)$. This was apparent when $P(Q)$ and $I(Q)$ were plotted together.
```

Reply:
Have changed the note to read:

![image](https://user-images.githubusercontent.com/10938679/140299194-a92efa5b-1148-4f1a-bfff-746c649c6686.png)

'here' now just resolves to https://github.com/SasView/sasview/issues/1698